db/migrations/20230509152120_village_reputation.php

```diff
@@ -0,0 +1,25 @@
+<?php
+declare(strict_types=1);
+
+use Phinx\Migration\AbstractMigration;
+
+final class VillageReputation extends AbstractMigration
+{
+    /**
+     * Change Method.
+     *
+     * Write your reversible migrations using this method.
+     *
+     * More information on writing migrations is available here:
+     * https://book.cakephp.org/phinx/0/en/migrations.html#the-change-method
+     *
+     * Remember to call "create()" or "update()" and NOT "save()" when working
+     * with the Table class.
+     */
+    public function change(): void
+    {
+        $this->execute("ALTER TABLE `users` ADD `village_rep` INT NOT NULL DEFAULT 0;");
+        $this->execute("UPDATE `users` SET `clan_office` = 0");
+        $this->execute("UPDATE `clans` SET `leader`=0, `elder_1`=0, `elder_2`=0");
```

Comment:
Let's remove this per not wanting to hard kick current clan officers immediately

Reply:
Will do, hadn't updated that after our discussion. I'll hard code it to be allow ~20 days after launch.